datatypes/src/util/mod.rs

```diff
@@ -3,5 +3,5 @@ pub mod helpers;
 pub mod ranges;
 mod result;
 pub mod test;
-
 pub use result::Result;
+pub mod time_step_iter;
```

Comment:
I would add this to the TimeStep type since it is bound to it.

Reply:
6bfbc3599f3a17ca18f7c7c5c8d10c1a11d53728